### DISCUSSION TITLE
Undefined property: RTippin\Messenger\Messenger::$headers".

### DISCUSSION THREAD

**Discussion by Forah** [RTippin/messenger#59] [Q&A]
Please could someone tell me why am getting this error? "Undefined property: RTippin\Messenger\Messenger::$headers". 
This is my User model
<?php
namespace App\Models;

use Illuminate\Database\Eloquent\Builder;
use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Foundation\Auth\User as Authenticatable;
use RTippin\Messenger\Contracts\MessengerProvider;
use RTippin\Messenger\Traits\Messageable;
 use RTippin\Messenger\Traits\Search;
 use Illuminate\Notifications\Notifiable;
use Laravel\Sanctum\HasApiTokens;
use Illuminate\Contracts\Auth\MustVerifyEmail;
use RTippin\Messenger\Messenger;
use RTippin\Messenger\Facades\Messenger;

class User extends Authenticatable implements MessengerProvider

{
    use HasApiTokens, HasFactory,Messageable,Search ;

**Comment by RTippin** [comment]
@Forah Do you have a full stack trace you can share? 

> This is my User model

I do not see any shared code.
